src/vscode-dts/vscode.proposed.languageModelThinkingPart.d.ts

```diff
@@ -16,7 +16,7 @@ declare module 'vscode' {
 		/**
 		 * The thinking/reasoning text content.
 		 */
-		value: string;
+		value: string | string[];
```

Comment:
Why `string[]`?

Reply:
Oh, interesting. And it would stream them in parallel like that? Seems very odd but ok.